### PR TITLE
chore: In `coder templates edit` change `--max_ttl` to `--max-ttl` and `--min_autostart_interval` to `--min-autostart-interval`,  as normally underscores are not used in command line argument names.

### PR DESCRIPTION
**Pull request by matifali**
As usually, underscores are not used in command line argument names.

### CODE REVIEW

File: coderd/database/migrations/000021_template_autobuild_constraints.down.sql [hunk 1]
```diff
@@ -1,2 +1,2 @@
-ALTER TABLE ONLY templates DROP COLUMN IF EXISTS max_ttl;
-ALTER TABLE ONLY templates DROP COLUMN IF EXISTS min_autostart_interval;
```
Comment (Emyrk) [inline coderd/database/migrations/000021_template_autobuild_constraints.down.sql:2]: Did you mean to change the database column names?

File: coderd/database/models.go [hunk 2]
```diff
@@ -465,8 +465,8 @@ type Template struct {
 	Provisioner          ProvisionerType `db:"provisioner" json:"provisioner"`
 	ActiveVersionID      uuid.UUID       `db:"active_version_id" json:"active_version_id"`
 	Description          string          `db:"description" json:"description"`
-	MaxTtl               int64           `db:"max_ttl" json:"max_ttl"`
-	MinAutostartInterval int64           `db:"min_autostart_interval" json:"min_autostart_interval"`
+	MaxTtl               int64           `db:"max-ttl" json:"max-ttl"`
+	MinAutostartInterval int64           `db:"min-autostart-interval" json:"min-autostart-interval"`
```
Comment (Emyrk) [inline coderd/database/models.go:469]: I think the json tags names are ok with `_`.

**Review by Emyrk — CHANGES_REQUESTED** [review]
Did you mean just to change the cli flag names? I agree with you on the cli.

File: coderd/database/migrations/000021_template_autobuild_constraints.down.sql (hunk 1 above)
Comment (matifali) [reply coderd/database/migrations/000021_template_autobuild_constraints.down.sql:2]: Yes, I mean to only change the CLI flags.

Comment (matifali) [reply coderd/database/migrations/000021_template_autobuild_constraints.down.sql:2]: But the change in the database can also be done to avoid confusion.

Comment (Emyrk) [reply coderd/database/migrations/000021_template_autobuild_constraints.down.sql:2]: We cannot edit migrations, you have to make a new one to rename a column. That being said, our other columns use `_`.

Can you make a new commit to only affect the cli flags?

**Comment by matifali** [comment]
Ok. I will update.

File: coderd/database/migrations/000021_template_autobuild_constraints.down.sql (hunk 1 above)
Comment (matifali) [reply coderd/database/migrations/000021_template_autobuild_constraints.down.sql:2]: ok

**Comment by matifali** [comment]
Closing this will create a new one.

**Pull request closed without merging**